doc/experiment.rst

```diff
@@ -259,4 +259,67 @@ the following:
 
   cd /scratch/smartsim/Training-Run ; /usr/bin/srun --output /scratch/smartsim/Training-Run/Training-Ensemble_3.out --error /scratch/smartsim/Training-Ensemble_3.err --job-name Training-Ensemble_3-CHTN0UI2TRE7 --nodes=1 --ntasks=24 /scratch/pyenvs/smartsim/bin/python ./train-model.py --LR=0.35000000000000003 &
 
-  wait
\ No newline at end of file
+  wait
+
+Prefixing Keys in the Orchestrator
+----------------------------------
+
+If each of multiple ensemble members attempt to use the same code access their respective models in
+the Orchestrator, the keys by which they do this will overlap and they can end up accessing each
+others' models inadvertently. To prevent this situation, the SmartSim Entity object supports key
+prefixing, which automatically prepends the name of the model to the keys by which it is accessed.
+With this enabled, key overlapping is no longer an issue and ensemble members can use the same code.
+
+Under the hood, calling ensemble.enable_key_prefixing() causes the SSKEYOUT environment variable to
+be set, which in turn causes all keys generated by an ensemble member to be prefixed with its model
+name.  Similarly, if the model for the ensemble member has incoming entities, the SSKEYIN environment
```

Comment:
Maybe "Similarly, if the model for the ensemble member has incoming entities, the SSKEYIN environment variable will be set and the keys associated with those inputs will be automatically prefixed."  would be more instructive as "Similarly, if the model for the ensemble member has incoming entities set via ``model.register_incoming_entity`` or ``ensemble.register_incoming_entity()``, the SSKEYIN environment variable will be set and the keys associated with those inputs will be automatically prefixed.

Reply:
Thanks, that will be much better for users.